src/txn/txn.c

```diff
@@ -536,12 +536,46 @@ __txn_config_operation_timeout(WT_SESSION_IMPL *session, const char *cfg[], bool
     return (0);
 }
 
+/*
+ * __txn_conf_operation_timeout --
+ *     Configure a transactions operation timeout duration.
+ */
+static int
+__txn_conf_operation_timeout(WT_SESSION_IMPL *session, const WT_CONF *conf, bool start_timer)
```

Comment:
Another thread that I'll hang off this new function because it's a convenient spot - the convention seems to be "old style -> `cfg`/`config`, new style -> `conf`", at least as a rule of thumb. I know discussing naming on a PR is a rabbit hole, but I suspect I'm going to find it tricky to keep this in my head and differentiate them consistently. Have you considered a name that more clearly says "precompiled", e.g. `pconf`?

Reply:
I'm okay with any alternate naming.